package.json

```diff
@@ -31,7 +31,8 @@
     "rebuild": "npm run clean && npm i && npm run build",
     "precommit:secrets": "gitleaks git --pre-commit --redact --staged --verbose --config .gitleaks/config.toml",
     "precommit:lint": "node_modules/.bin/lint-staged",
-    "precommit:verify": "npm run typecheck"
+    "precommit:verify": "npm run typecheck",
+    "setup": "npm ci && node run hmpps-allow-scripts"
```

Comment:
developers and ci would need to run this instead of the npm install in order to ensure appropriate scripts are installed.
If they forget then new dependencies would be pulled in but the .npmrc config should ensure no new dependencies are run